boards/px4/fmu-v4/init/rc.board_defaults

```diff
@@ -0,0 +1,27 @@
+#!nsh
+#
+# PX4 FMUv4 specific board init
+#
+#------------------------------------------------------------------------------
+#
+# UART mapping on FMUv5:
+#
+# UART1         /dev/ttyS0      wifi
+# USART2        /dev/ttyS1      TELEM1 (flow control)
+# USART3        /dev/ttyS2      TELEM2 (flow control)
+# UART4
+# UART7                         CONSOLE
+# UART8         /dev/ttyS6      SERIAL4/TELEM4
+#
+#------------------------------------------------------------------------------
+#
+
+if [ $AUTOCNF = yes ]
+then
+	# Disable safety switch by default
+	param set CBRK_IO_SAFETY 22027
+
+	param set SYS_FMU_TASK 1
+fi
+
+set MIXER_AUX none
```

Comment:
If you move this to `rc.sensors` we can remove https://github.com/PX4/Firmware/blob/master/ROMFS/px4fmu_common/init.d/rc.interface#L28.

Reply:
I was hoping to untangle it a little bit more so that it can be mostly dropped. Something like the default for MOUNT_AUX remaining NONE, then perhaps split the entire rc.interface AUX section into a separate script that's only executed if the AUX device and AUX file exist.

Anyway, not sure yet, there are other awkward side effects to think about. For example vmount settings overriding the AUX mixer. This could even be dangerous in a VTOL setup.